cli/config/config.go

```diff
@@ -102,11 +101,17 @@ func GetSeedImage() string {
 
 func GetPackageName() string {
 	metadata := GetMetaData()
+	prefix := PackagePrefix
+	suffix := "tar.zst"
+
+	if IsZarfInitConfig() {
+		return fmt.Sprintf("zarf-init-%s.tar.zst", GetArch())
+	}
+
 	if metadata.Uncompressed {
-		return PackagePrefix + metadata.Name + ".tar"
-	} else {
-		return PackagePrefix + metadata.Name + ".tar.zst"
+		suffix = "tar"
 	}
+	return fmt.Sprintf("%s-%s-%s.%s", prefix, metadata.Name, GetArch(), suffix)
```

Comment:
I don't think there's an official check to make enforce `metadata.Name` to be spaceless. What do you think about doing a strings.trim() or a replace and switch the spaces with a - character?

Reply:
I changed this because I added a validator in the last commit.  It felt cleaner to be explicit at package create time.

Reply:
Would you look at that!